addons/diagnostic/fnc_projectileTracking_trackProjectile.sqf

```diff
@@ -39,13 +43,9 @@ if (!isNull _projectile) then {
     } else {
         _bulletData = [_initialProjectileData];
         _data = [_handle, _bulletData];
+        GVAR(projectileData) set [_index, _data];
     };
-
-    _bulletData pushBack [(getPos _projectile), vectorMagnitude (velocity _projectile)];
-
-    _data set [1, _bulletData];
-
-    GVAR(projectileData) set [_index, _data];
```

Comment:
same here? outside of the if block?

Reply:
These are not needed, because only commands that modify the array are used. Previously the function set a sub array, and a sub sub array to exactly what they were before.